read-fonts/src/tables/cmap.rs

```diff
@@ -39,6 +40,22 @@ impl<'a> Cmap<'a> {
         }
         None
     }
+
```

Comment:
So there is already a trait defined for this in GSUB, and we should make the signatures match. That implementation is older than `IntSet`, though, so let's just open an issue to fix this later?

Reply:
Done, added a new issue.